### ISSUE TITLE
Blockchain explorer - Coin balance history tab, doesn't show most recent transaction/balance

### ISSUE THREAD

**Issue by vman46** [electroneum/electroneum-sc-block-explorer#17]
Coin balance history tab on the ETN blockchain explorer, doesn't always show the most recent transaction/balance. Below are two wallets that don't show current value. and third wallet that does show it correctly.

coin balance history tab shows 50 etn as most recent trans, but current value 24.99 ETN
https://blockexplorer.electroneum.com/address/0xe382f6d700Ec5C0971AEC4876C1e020a127979A5/coin-balances#address-tabs

coin balance history tab shows 1,702 ETN as most recent trans, but current value 1,727 etn
https://blockexplorer.electroneum.com/address/0x2FF09EE69D5B9c9347eA32B4f1d8597D3F201841

this one displays correctly
coin balance history tab shows 173.149 as most recent transaction, and current balance is 173.149
https://blockexplorer.electroneum.com/address/0x19031FaE16F670bb673a8E02aA502bE21F75CA25/coin-balances#address-tabs

### Environment
* Elixir Version: 1.12.2
* Erlang Version: 24
* Explorer Version: 

* User Agent: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36`

Chrome

*Tell us how to reproduce this issue. If possible, push up a branch to your fork with a regression test we can run to reproduce locally.*

Basically go to the links I provided above and see that the coin balance history tab for various wallets doesn't always show correct information.

*Tell us what should happen.*

### Actual Behaviour

*Tell us what happens instead.*
